examples/websocket_client.py

```
@@ -0,0 +1,281 @@
+            "ref": 4
+        }
+        
+        await self.websocket.send(json.dumps(message))
+        
+        return {
+            "subscription_id": self.subscription_id,
+            "subscribed_systems": list(self.subscriptions),
+            "connected": self.running
+        }
+    
+    async def disconnect(self):
```

Ruff (0.11.9)

Missing return type annotation for public function `disconnect`

Add return type annotation: `None`

(ANN201)